src/imitation/util/networks.py

```diff
@@ -117,6 +117,57 @@ def forward(self, x: th.Tensor) -> th.Tensor:
         return (x - self.running_mean) / th.sqrt(self.running_var + self.eps)
 
 
+class EMARunningNorm(RunningNorm):
```

Comment:
I weakly lean towards making `RunningNorm` an abstract class whose children must override `update_stats`, and then have concrete implementations `EMARunningNorm` and `SimpleRunningNorm` (currently `RunningNorm`).

Main reason is that it feels wrong to say that `EMARunningNorm` is-a simple moving average (which is what `RunningNorm` implements), and we don't use any of `RunningNorm.update_stats` but override it fully.

Reply:
What do you think of the class structure being
```
class BaseNorm(nn.Module, ABC)
class RunningNorm(BaseNorm)
class EMANorm(BaseNorm)
```
This would not break the public API and does not seem too awkward.

Reply:
I'm not too worried about breaking changes (we're not v1.0 yet, and `RunningNorm` was only recently introduced). But `BaseNorm` seems like a reasonable name for it, I'm happy with this hierarchy if it seems more natural to you.

Reply:
Awesome! It's pushed.